.github/workflows/ci.yml

```diff
@@ -14,10 +14,10 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [django22, quality, django30, django31, django32]
+        toxenv: [quality, django32, django40]
         exclude:
-          - python-version: '3.8'
-            toxenv: quality
+        - python-version: '3.8'
```

Comment:
This exclude section was added for `Python 3.5` at some point but mistakenly got updated to `3.8` with a cleanup Job. Please remove this exclude section and fix the new quality failures as well.